airbyte-cdk/bulk/toolkits/load-http/build.gradle

```diff
@@ -2,10 +2,8 @@ dependencies {
     implementation project(':airbyte-cdk:bulk:core:bulk-cdk-core-base')
     implementation project(':airbyte-cdk:bulk:core:bulk-cdk-core-load')
 
-    api("org.apache.commons:commons-csv:1.11.0")
-
-    implementation 'com.squareup.okhttp3:okhttp:4.12.0'
-    implementation 'dev.failsafe:failsafe-okhttp:3.3.2'
+    api 'com.squareup.okhttp3:okhttp:4.12.0'
+    api 'dev.failsafe:failsafe-okhttp:3.3.2'
```

Comment:
I changed it for `api` because most of the users of the lib will have to instantiate a OkHttpClient